src/main/java/com/iexec/core/task/TaskService.java

```diff
@@ -110,7 +111,16 @@ public TaskService(TaskRepository taskRepository) {
     }
 
     public List<Task> getInitializedOrRunningTasks() {
-        return taskRepository.findByCurrentStatus(Arrays.asList(INITIALIZED, RUNNING),
+        return getInitializedOrRunningTasks(false);
+    }
+
+    public List<Task> getInitializedOrRunningTasks(boolean onlyStandardTasks) {
```

Comment:
`onlyStandardTasks` should fetch "any but not 0x0..01"
Else "any" is fetched

Reply:
To be clearer, with current implementation, a TEE-enabled worker will never be able to compute standard tasks (but it should).

Reply:
Oh, yes, you're right. My bad!

Reply:
Thanks! Do you think `boolean onlyStandardTasks` -> `boolean shouldExcludeTeeTasks` is better?

Reply:
That may be better, yes. It represents the operation in a better way. Thanks for the idea!